schedule/functional/boot_into_snapshot_after_upgrade.yaml

```diff
@@ -0,0 +1,22 @@
+---
+name: boot_into_snapshot_after_upgrade
```

Comment:
In ticket, AC1 states: `boot_into_snapshot is scheduled at the end of `upgrade_Leap_15.1_gnome` scenario (Leap->Tumbleweed) for the last Leap version.`

There is no verification run for test suite `upgrade_Leap_15.1_gnome`.
And the creation of test suite `boot_into_snapshot_after_upgrade` is invalid.